composer.json

```
@@ -0,0 +1,9 @@
+{
+	"scripts": {
+		"cs:check": "php-cs-fixer fix --dry-run",
```

```suggestion
		"cs:check": "php-cs-fixer fix --dry-run --diff",
```
https://github.com/nextcloud/server/pull/20502